src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceProgress.cs

```diff
@@ -166,6 +213,10 @@ class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInt
 
         private ProgressPane _progPane = null;
         private PendingProgress _pendingProgress = null;
+        // The timer set up 'progPaneUpdateFlag' every 'UpdateTimerThreshold' milliseconds to update 'ProgressPane'
+        private Timer _progPaneUpdateTimer;
+        private const int UpdateTimerThreshold = 200;
+        private bool progPaneUpdateFlag;
```

Comment:
`progPaneUpdateFlag` needs to be changed to an `int` field if we want to use `InterLock.CompareAndExchange`.